test.py

```diff
@@ -53,14 +53,6 @@ def test_manager(self):
         self.assertEqual(r, "jaja-lol-a")
 
         txt = 'jaja---lol-méméméoo--a'
-        r = slugify(txt, max_length=19, word_boundary=True)
```

Comment:
What you can do if you find a bug, put a fix in and submit it separately. New params for new features also should be on their own. This way, minimally, the bug is going to be merged upstream. I think len=17 vs. len=19 could be a bug.